### PR TITLE
adding delist portal link

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2530

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/fix-error-code-550-5-7-1-in-exchange-online.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ In this cases, the NDR the sender receives would include information in the **Di
 
    `5.7.1 Service unavailable; Client host [xxx.xxx.xxx.xxx] blocked using Blocklist 1; To request removal from this list please forward this message to delist@messaging.microsoft.com`
 
-To remove the restriction on the sender's source email system, forward the NDR message to delist@messaging.microsoft.com.
+To remove the restriction on the sender's source email system, forward the NDR message to delist@messaging.microsoft.com and please also see [Use the delist portal to remove yourself from the blocked senders list](/microsoft-365/security/office-365-security/use-the-delist-portal-to-remove-yourself-from-the-office-365-blocked-senders-lis).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/non-delivery-reports-in-exchange-online/fix-error-code-550-5-7-1-in-exchange-online.md:133]:
```suggestion
To remove the restriction on the sender's source email system, forward the NDR message to delist@messaging.microsoft.com. Also see [Use the delist portal to remove yourself from the blocked senders list](/microsoft-365/security/office-365-security/use-the-delist-portal-to-remove-yourself-from-the-office-365-blocked-senders-lis).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. For information, in accordance with the Microsoft Style Guide, we should not use the word "please" unless the customer is asked to do something that is inconvenient or the application or site is to blame for the situation. Many thanks.

**Comment by get-itips** [comment]
> Copy review. For information, in accordance with the Microsoft Style Guide, we should not use the word "please" unless the customer is asked to do something that is inconvenient or the application or site is to blame for the situation. Many thanks.

good to know!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by spp2** [comment]
Hi, 
please note that the added URL is incomplete, giving an error 404; 
and that the address delist@messaging.microsoft.com is still "blocked by an administrator on frondbridge.com", rendering this step useless .
It includes this text:

> NOTE: For issues sending to Office 365, please include an NDR or bounce message. https://aka.ms/o365snds

I see no point in keeping delist@messaging.microsoft.com in this help page.

**Comment by get-itips** [comment]
> Hi,
> please note that the added URL is incomplete, giving an error 404;

the URL will not work in the PR but in the published version of the article (it will take time to sync)

> and that the address [delist@messaging.microsoft.com](mailto:delist@messaging.microsoft.com) is still "blocked by an administrator on frondbridge.com", rendering this step useless .

thanks for including this info.

**Comment by spp2** [comment]
> the URL will not work in the PR but in the published version of the article (it will take time to sync)

OK confirmed, thanks: 
I was concerned by the seemingly truncated URL, but it is correct actually.

**Pull request merged**